pom.xml

```diff
@@ -1234,6 +1234,8 @@
                 <module>samples/client/petstore/typescript-angular-v12-provided-in-root</module>
                 <module>samples/client/petstore/typescript-angular-v13-provided-in-root</module>
                 <module>samples/client/petstore/typescript-angular-v14-provided-in-root</module>
+                <module>samples/client/petstore/typescript-angular-v15-provided-in-root</module>
```

Comment:
in order for this to work, you need to copy over the configuration and tests from https://github.com/OpenAPITools/openapi-generator/blob/master/samples/client/petstore/typescript-angular-v14-provided-in-root/pom.xml and https://github.com/OpenAPITools/openapi-generator/tree/master/samples/client/petstore/typescript-angular-v14-provided-in-root/tests/default/src

Reply:
Done, I had to slightly adapt the code to avoid deprecation warnings.